### ISSUE TITLE
Validation after action

### ISSUE THREAD

**Issue by leovieira20** [mlykotom/valifi#12]
Hey @mlykotom , congrats for the library, I'm willing to use it on my project but I don't want the "on-the-fly" validation, I think it's not the best UX experience for my project at least.

Are you considering having this kind of option for the library?

Cheers,
Leo

**Comment by mlykotom** [comment]
So you would like to not show the error and show it manually from code (e.g. when clicking submit)?
Or explain little bit more please.

**Comment by leovieira20** [comment]
Yes, the behavior I'm looking for is: let the user do his thing and then I'll tell if it's valid or not.

I've seeing this type of behavior in many systems (Google, Facebook to mention a few), I'm not sure if it's the best UX practice to show **some** errors while typing, it's too much information at the same time you know?

To put it another way, I would like to kickoff validation manually, so the messages would be shown post action and not while action

**Comment by mlykotom** [comment]
Just so you know. Since version 1.0.0 the error is actually delayed after user stops typing the input.
It's configurable, so you can use how much time do you want.

```java
ValiFi.install(this, new ValiFi.Builder()
		.setErrorDelay(1000)
		.build()
);
```

I'll extend it so that the error may never be shown as you mentioned.

**Comment by leovieira20** [comment]
@mlykotom thanks for the heads up, i was checking the docs and saw that option

ok, that would be great, thanks!

**Comment by mlykotom** [comment]
Resolved in 1.0.2. 
See [Wiki - Manual Validation](https://github.com/mlykotom/valifi/wiki/Manual-Validation) for more info.
